# A set of fiducial to fixed frame transforms
BoxTransform[] transforms
